package.json

```diff
@@ -41,7 +42,7 @@
     "graphql": "^16.6.0",
     "imask": "^6.4.3",
     "jotai": "^1.9.1",
-    "next": "^13.0.2",
+    "next": "^13.0.5",
```

Comment:
We should probably roll this back because this is only a documentation task.

Reply:
Fixes a bug in Next.js.